### PR TITLE
fetch post-GA corrections and additions

### PR DESCRIPTION
**Pull request by taroface**
DOC-10372

### CODE REVIEW

File: src/current/v23.2/molt-fetch.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ Complete the following items before using MOLT Fetch:
 
 - If you plan to use continuous replication (using either [`--ongoing-replication`](#replication) or the [CDC cursor](#cdc-cursor)):
 
+	- The source and target database names **must** be identical when using `--ongoing-replication`.
```
[the remark this answers is not in the document]
Comment (ryanluu12345) [reply src/current/v23.2/molt-fetch.md:48]: Are those changes merged in and released in the latest `replicator` binary?

[the remark this answers is not in the document]
Comment (sravotto) [reply src/current/v23.2/molt-fetch.md:48]: The latest version of replicator allows you to have a different database name for the target.

[the remark this answers is not in the document]
Comment (ryanluu12345) [reply src/current/v23.2/molt-fetch.md:48]: @taroface , which means we can probably remove this requirement and doc now.

File: src/current/v23.2/molt-fetch.md [hunk 2]
```diff
@@ -173,44 +175,45 @@ To verify that your connections and configuration work properly, run MOLT Fetch
...
+| `--flush-size`                                | Size (in bytes) before the source data is flushed to intermediate files. **Note:** If `--flush-rows` is also specified, the fetch behavior is based on the flag whose criterion is met first.                                                                                                                                                                                                                                                                                                                                                  |
+| `--import-batch-size`                         | The number of files to be imported at a time to the target database. This applies only when using the [`IMPORT INTO` mode](#fetch-mode) for loading data into the target. **Note:** Increasing this value can improve fetch performance, but very high values are not recommended. If any one file in the import batch fails, you will need to [retry](#fetch-continuation) the entire batch.<br><br>**Default:** `10`                                                                                                                         |
```
Comment (ryanluu12345) [inline src/current/v23.2/molt-fetch.md:195]: Default is actually `1000` now.

Comment (ryanluu12345) [reply src/current/v23.2/molt-fetch.md:195]: @ZhouXing19 to add her review here.

File: src/current/v23.2/molt-fetch.md [hunk 3]
```diff
@@ -344,11 +351,11 @@ By default, MOLT Fetch moves all data from the [`--source`](#source-and-target-d
 --schema-filter 'public'
 ~~~
 
-`--table-filter` specifies a range of tables to move to CockroachDB, based on a regex string. For example, to move every table in the source database that has "user" in the title:
+`--table-filter` specifies a range of tables to move to CockroachDB, based on a POSIX regex string. For example, to move every table in the source database that has "user" in the title:
 
 {% include_cached copy-clipboard.html %}
 ~~~
---table-filter '\w*user\w*'
+--table-filter '.*user.*'
```
Comment (ryanluu12345) [inline src/current/v23.2/molt-fetch.md:358]: Awesome, thanks for fixing this example!

File: src/current/v23.2/molt-fetch.md [hunk 4]
```diff
@@ -561,7 +568,7 @@ When the `--ongoing-replication` flag is set, changes on the source database are
 
 Before using this feature, complete the following:
 
-- Install the Replicator binary. Before running `molt fetch` with continuous replication, [download the binary that matches your system](https://github.com/cockroachdb/replicator/wiki/Installing#automated-builds). The Replicator binary **must** be located in the same directory as your [`molt` binary](#installation).
+- Ensure that the source and target database names are **identical**. Otherwise, replication will not work.
```
Comment (ryanluu12345) [inline src/current/v23.2/molt-fetch.md:571]: No longer needed (reference Silvano's comment at the top)

File: src/current/v24.1/molt-fetch.md [hunk 5]
```diff
@@ -45,6 +45,8 @@ Complete the following items before using MOLT Fetch:
 
 - If you plan to use continuous replication (using either [`--ongoing-replication`](#replication) or the [CDC cursor](#cdc-cursor)):
 
+	- The source and target database names **must** be identical when using `--ongoing-replication`.
```
Comment (ryanluu12345) [inline src/current/v24.1/molt-fetch.md:48]: Ditto

File: src/current/v24.1/molt-fetch.md [hunk 6]
```diff
@@ -173,44 +175,45 @@ To verify that your connections and configuration work properly, run MOLT Fetch
...
+| `--flush-size`                                | Size (in bytes) before the source data is flushed to intermediate files. **Note:** If `--flush-rows` is also specified, the fetch behavior is based on the flag whose criterion is met first.                                                                                                                                                                                                                                                                                                                                                  |
+| `--import-batch-size`                         | The number of files to be imported at a time to the target database. This applies only when using the [`IMPORT INTO` mode](#fetch-mode) for loading data into the target. **Note:** Increasing this value can improve fetch performance, but very high values are not recommended. If any one file in the import batch fails, you will need to [retry](#fetch-continuation) the entire batch.<br><br>**Default:** `10`                                                                                                                         |
```
Comment (ryanluu12345) [inline src/current/v24.1/molt-fetch.md:195]: Default is still `1000`

File: src/current/v24.1/molt-fetch.md [hunk 7]
```diff
@@ -561,7 +568,7 @@ When the `--ongoing-replication` flag is set, changes on the source database are
 
 Before using this feature, complete the following:
 
-- Install the Replicator binary. Before running `molt fetch` with continuous replication, [download the binary that matches your system](https://github.com/cockroachdb/replicator/wiki/Installing#automated-builds). The Replicator binary **must** be located in the same directory as your [`molt` binary](#installation).
+- Ensure that the source and target database names are **identical**. Otherwise, replication will not work.
```
Comment (ryanluu12345) [inline src/current/v24.1/molt-fetch.md:571]: Ditto

**Review by ryanluu12345 — APPROVED** [review]
Nits only: once fixed, merge

File: src/current/v23.2/molt-fetch.md (hunk 2 above)
Comment (ZhouXing19) [inline src/current/v23.2/molt-fetch.md:195]:
```suggestion
| `--import-batch-size`                         | The number of files to be imported at a time to the target database. This applies only when using the [`IMPORT INTO` mode](#fetch-mode) for loading data into the target. **Note:** Increasing this value can improve the performance of full-scan queries on the target database shortly after the fetch process finishes, but very high values are not recommended. If any one file in the import batch fails, you will need to [retry](#fetch-continuation) the entire batch.<br><br>**Default:** `10`                                                                                                                         |
```

File: src/current/v23.2/molt-fetch.md [hunk 8]
```diff
@@ -173,44 +173,45 @@ To verify that your connections and configuration work properly, run MOLT Fetch
...
+| `--flush-size`                                | Size (in bytes) before the source data is flushed to intermediate files. **Note:** If `--flush-rows` is also specified, the fetch behavior is based on the flag whose criterion is met first.                                                                                                                                                                                                                                                                                                                                                  |
+| `--import-batch-size`                         | The number of files to be imported at a time to the target database. This applies only when using the [`IMPORT INTO` mode](#fetch-mode) for loading data into the target. **Note:** Increasing this value can improve the performance of full-scan queries on the target database shortly after fetch completes, but very high values are not recommended. If any one file in the import batch fails, you will need to [retry](#fetch-continuation) the entire batch.<br><br>**Default:** `1000`                                               |
```
Comment (mdlinville) [inline src/current/v23.2/molt-fetch.md:193]:
```suggestion
| `--import-batch-size`                         | The number of files to be imported at a time to the target database. This applies only when using the [`IMPORT INTO` mode](#fetch-mode) for loading data into the target. **Note:** Increasing this value can improve the performance of full-scan queries on the target database shortly after a fetch completes, but very high values are not recommended. If any individual file in the import batch fails, you must [retry](#fetch-continuation) the entire batch.<br><br>**Default:** `1000`                                               |
```

File: src/current/v23.2/molt-fetch.md [hunk 9]
```diff
@@ -173,44 +173,45 @@ To verify that your connections and configuration work properly, run MOLT Fetch
...
+| `--direct-copy`                               | Enables [direct copy mode](#fetch-mode), which copies data directly from source to target without using an intermediate store.                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `--export-concurrency`                        | Number of shards to export at a time, each on a dedicated thread. This only applies when exporting data from the source database, not when loading data into the target database. **Note:** The number of concurrent threads is the product of `--export-concurrency` and `--table-concurrency`. See [Best practices](#best-practices).<br><br>This value **cannot** be set higher than `1` when moving data from MySQL. Refer to [Best practices](#best-practices).<br><br>**Default:** `4` with a PostgreSQL source; `1` with a MySQL source |
```
Comment (mdlinville) [inline src/current/v23.2/molt-fetch.md:189]: Do you need two links to the Best Practices section in such close proximity?

File: src/current/v23.2/molt-fetch.md [hunk 10]
```diff
@@ -173,44 +173,45 @@ To verify that your connections and configuration work properly, run MOLT Fetch
...
+| `--local-path`                                | The path within the [local file server](#local-file-server) where intermediate files are written (e.g., `data/migration/cockroach`). `--local-path-listen-addr` must be specified.                                                                                                                                                                                                                                                                                                                                                             |
+| `--local-path-crdb-access-addr`               | Address of a [local file server](#local-file-server) that is reachable by CockroachDB. This flag is only necessary if CockroachDB cannot reach the local address specified with `--local-path-listen-addr` (e.g., when moving data to a CockroachDB {{ site.data.products.cloud }} deployment).<br><br>**Default:** Value of `--local-path-listen-addr`. `--local-path` and `--local-path-listen-addr` must be specified.                                                                                                                      |
```
Comment (mdlinville) [inline src/current/v23.2/molt-fetch.md:195]: I don't really understand what the Default bit means.

File: src/current/v23.2/molt-fetch.md [hunk 11]
```diff
@@ -344,11 +345,11 @@ By default, MOLT Fetch moves all data from the [`--source`](#source-and-target-d
 --schema-filter 'public'
 ~~~
 
-`--table-filter` specifies a range of tables to move to CockroachDB, based on a regex string. For example, to move every table in the source database that has "user" in the title:
+`--table-filter` specifies a range of tables to move to CockroachDB, based on a POSIX regex string. For example, to move every table in the source database that has "user" in the title:
```
Comment (mdlinville) [inline src/current/v23.2/molt-fetch.md:348]:
```suggestion
`--table-filter` specifies a range of tables to move to CockroachDB,formatted as a POSIX regex string. For example, to move every table in the source database that has "user" in the title:
```

**Review by mdlinville — APPROVED** [review]
Same feedback for v24.1 versions of the files. :)

File: src/current/v23.2/molt-fetch.md (hunk 9 above)
Comment (taroface) [reply src/current/v23.2/molt-fetch.md:189]: Oops! Good catch

**Comment by taroface** [comment]
TFTRs! Addressed and merging.

**Pull request merged**